app/config/global_view_parameters.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
+    help_url:
+        en: "https://wiki.surfnet.nl/display/conextsupport/Profile+page"
+        nl: "https://wiki.surfnet.nl/display/conextsupport/Profile+page"
```

Comment:
What about application name?

Could you try adding a `global_view_parameters.yml.dist` and letting incenteev/parameter-handler handle creation of the YAML file? This means other deploys of OpenConext-profile don't have to change files under version control. [Managing multiple ignored files](https://github.com/Incenteev/ParameterHandler/blob/f7b677a5f9bfd8bf5f165a0ff96ce7426b870312/README.md#managing-multiple-ignored-files)